### PR TITLE
[Stream] Added advanced player properties content

### PR DESCRIPTION
**Pull request by dcpena**
Addresses PCX-4509.

### CODE REVIEW

**Comment by Schachte** [comment]
Here are some examples to persist via API using curl

> No additional work is required if you're using the Cloudflare player. See at the bottom for customers using their own custom player. 

## Example 1
### All fields
```bash
 curl --location --request POST "https://api.cloudflare.com/client/v4/accounts/$ACCOUNT_ID/stream/$VIDEO_UID" \
--header "Authorization: Bearer $SECRET" \
--header 'Content-Type: application/json' \
--data-raw '{
    "publicDetails": {
        "title": "Optional video title",
        "share_link": "https://my-cool-share-link.cloudflare.com",
        "channel_link": "https://www.cloudflare.com/products/cloudflare-stream/",
        "logo": "https://upload.wikimedia.org/wikipedia/commons/thumb/9/94/Cloudflare_Logo.png/480px-Cloudflare_Logo.png"
    }
}' | jq ".result.publicDetails"
```

## Example 2
### Shows how fields can be optional
```bash
curl --location --request POST "https://api.cloudflare.com/client/v4/accounts/$ACCOUNT_ID/stream/$VIDEO_UID" \
--header "Authorization: Bearer $SECRET" \
--header 'Content-Type: application/json' \
--data-raw '{
    "publicDetails": {
        "logo": "https://upload.wikimedia.org/wikipedia/commons/thumb/9/94/Cloudflare_Logo.png/480px-Cloudflare_Logo.png"
    }
}'
```

If you're using your own player, you can pull the JSON from this endpoint:

```
https://customer-<ID>.cloudflarestream.com/<VIDEO_ID>/metadata/playerEnhancementInfo.json
```

**Review by mbetz08 — APPROVED** [review]
Looks great!

File: content/stream/edit-videos/player-enhancements.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Add player enhancements
+
+With player enhancements, you can modify your video player to incorporate elements of your branding, such as your logo, and customize additional options to present to your viewers.
+
+The player enhancements are automatically applied to videos using the Stream Player, but you will need to add the details via the `publicDetails` property when using your own player.
+
+## Properties
+
+- `title`: The title that appears when viewers hover over the video. The title can differ from the file name of the video. 
+- `share_link`: Provides the user with a "click-to-copy" option to easily share the video URL.
```
Comment (marciocloudflare) [inline content/stream/edit-videos/player-enhancements.md]:
```suggestion
- `share_link`: Provides the user with a click-to-copy option to easily share the video URL.
```

Comment (marciocloudflare) [reply content/stream/edit-videos/player-enhancements.md]: Suggestion to remove as we should avoid quotation marks in the docs.

File: content/stream/edit-videos/player-enhancements.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- `channel_link`: The HTTPS URL users will be directed to when selecting the logo from the video player. 
+- `logo`: A valid HTTPS URL for the image of your logo.
+
+## Customize your own player
+
+The example below includes every property you can set via `publicDetails`.
+
+```bash
+ curl --location --request POST "https://api.cloudflare.com/client/v4/accounts/<$ACCOUNT_ID>/stream/<$VIDEO_UID>" \
```
Comment (marciocloudflare) [inline content/stream/edit-videos/player-enhancements.md]:
```suggestion
curl --location --request POST "https://api.cloudflare.com/client/v4/accounts/<$ACCOUNT_ID>/stream/<$VIDEO_UID>" \
```

File: content/stream/edit-videos/player-enhancements.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+You can also pull the JSON by using the endpoint below.
+
+`https://customer-<ID>.cloudflarestream.com/<VIDEO_ID>/metadata/playerEnhancementInfo.json`
+
+## Update player properties via the Cloudflare dashboard
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
+2. Click **Stream** > **Videos**.
```
Comment (marciocloudflare) [inline content/stream/edit-videos/player-enhancements.md]:
```suggestion
2. Select **Stream** > **Videos**.
```

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**